src/Uno.Extensions.Navigation.UI/NavigatorExtensions.cs

```diff
@@ -48,16 +48,16 @@ public static class NavigatorExtensions
 	}
 #endif
 
-	public static async Task<NavigationResponse?> GoBack(this INavigator navigator, object sender)
+	private static async Task<INavigator?> GoBackNavigator(this INavigator navigator)
 	{
 		var dispatcher = navigator.Get<IServiceProvider>()!.GetRequiredService<IDispatcher>();
 
 		// Default to navigating back on the current navigator
 		if (await navigator.CanNavigate(new Route(Qualifiers.NavigateBack)))
-			//is ControlNavigator ctrlNav &&
-			//await dispatcher.ExecuteAsync(async ct => ctrlNav.CanGoBack))
+		//is ControlNavigator ctrlNav &&
+		//await dispatcher.ExecuteAsync(async ct => ctrlNav.CanGoBack))
```

Comment:
Still needed ?

Reply:
No - will do a clean up PR separately for this and to add documentation to navigation apis